templates/zerver/help/send-a-private-message.md

```diff
@@ -4,29 +4,35 @@ Zulip allows users to send private messages to each other.
 
 ## There are two ways to send a Private Message:
```

Comment:
You should rephrase this to "You can send a private message by through the compose box or the users list" and make sure it's not a header.